packages/common/src/hooks/chats/useSetInboxPermissions.ts

```diff
@@ -60,55 +58,40 @@ export const useSetInboxPermissions = ({
         )
         try {
           const sdk = await audiusSdk()
-          await sdk.chats.permit({ permit: permission })
-          setPermissionStatus(Status.SUCCESS)
+          permitList = transformMapToPermitList(permitMap)
+          await sdk.chats.permit({ permitList, allow: true })
+          doFetchPermissions()
+          setSavePermissionStatus(Status.SUCCESS)
```

Comment:
do we still need this? wonder if we could use isSubmitting instead which should be true while this onSubmit call is in progress

Reply:
i think you're right